lib/src/core/configs/text_field_config.dart

```diff
@@ -26,6 +26,9 @@ class TextFieldConfig extends ComponentConfig {
     required this.textSelectionTheme,
     required this.mouseCursor,
     required this.keyboardType,
+    required this.readOnly,
+    required this.obscureText,
+    required this.enabled,
```

Comment:
No need to add these in config, adding them to core widget should be enough.

Reply:
ok dai